WordPress/gutenberg
# Block Editor: Stabilize 'generateAnchors' setting

## What?
Part of #39977.

PR stabilizes the `generateAnchors` block editor setting, introduced in #38780.

## Why?
The setting was cherry-picked in 5.9.1, and I've not seen any bugs reports. It's also on/off settings; I don't expect significant changes.

## Testing Instructions
1. Open a Post or Page.
2. Add multiple headings.
3. Check that the anchor isn't automatically generated.
4. Enable feature.
5. Refresh the editor.
6. Confirm that anchors are generated.
7. They update as you change heading contents.
